src/CachePlugin.php

```diff
@@ -138,6 +137,34 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     }
 
     /**
```

Comment:
Calculate the timestamp when this cache item should be dropped from the cache. 

This can be higher than the HTTP max-age when using cache validation.